signer/plugin.go

```diff
@@ -21,12 +21,13 @@ import (
 // pluginSigner signs artifacts and generates signatures.
 // It implements notation.Signer
 type pluginSigner struct {
-	plugin       plugin.SignPlugin
-	keyID        string
-	pluginConfig map[string]string
+	plugin              plugin.SignPlugin
+	keyID               string
+	pluginConfig        map[string]string
+	manifestAnnotations map[string]string
```

Comment:
nit: can we rename it to `pluginAnnotations`? since it's annotations from the plugin.